crypto/x509/x509_req.c

```diff
@@ -227,13 +227,21 @@ X509_ATTRIBUTE *X509_REQ_get_attr(const X509_REQ *req, int loc)
 
 X509_ATTRIBUTE *X509_REQ_delete_attr(X509_REQ *req, int loc)
 {
-    return X509at_delete_attr(req->req_info.attributes, loc);
+    X509_ATTRIBUTE *attr = NULL;
+
+    attr = X509at_delete_attr(req->req_info.attributes, loc);
+    if (attr == NULL)
```

Comment:
No, as I wrote, must use `!= NULL` for checking success.

Reply:
I can't believe I did such a simple mistake. Sorry again.